src/fs.bash

```diff
@@ -30,9 +30,27 @@ fs.is_symlink() {
     return 1
 }
 
-# check whether file is a symlink
+# check whether file is symlinked to a package
 fs.is_ellipsis_symlink() {
-    if [[ -L "$1" && "$(readlink "$1")" == $ELLIPSIS_PATH/* ]]; then
+    fs.is_target_symlink "$1" "$ELLIPSIS_PACKAGES"
+}
+
+# check whether file is symlinked to ellipsis internal library
+fs.is_ellipsis_core_symlink() {
+    fs.is_target_symlink "$1" "$ELLIPSIS_PATH"
+}
+
+# check whether file is symlinked to target
+fs.is_target_symlink() {
+    if [[ -L "$1" && "$(readlink "$1")" == "$2"/* ]]; then
+        return 0
+    fi
+    return 1
+}
+
+# check whether file is symlinked to one package
+fs.is_package_symlink() {
+    if [[ -L "$1" && "$(readlink "$1")" == $ELLIPSIS_PACKAGES/* ]]; then
```

Comment:
Any reason you copied the old implementation instead of adopting the new `fs.is_target_symlink` function?

`fs.is_target_symlink "$1" "$ELLIPSIS_PACKAGES"`

Reply:
Nope you're right, I should have reused `fs.is_target_symlink`, it's a mistake from me, probably another failed merge on my side :p